lib/App/index.js

```diff
@@ -189,6 +189,25 @@ class App {
 						}
 					}
 				}
+
+                if (driver.zwave) {
+                    for (let j = 0; j < driver.settings.length; j++) {
+                        let setting = driver.settings[j];
+                        let signed;
+                        let size = setting.zwave.size;
```

Comment:
Thus might throw ‘size is undefined’ if a zwave driver implements a non zwave setting.